core/domain/feedback_services.py

```diff
@@ -487,7 +739,14 @@ def _send_instant_emails(
 
 def _send_feedback_thread_status_change_emails(
         recipient_list, feedback_message_reference, old_status, new_status):
-    """send email feedback thread status change."""
+    """Notifies the given recipients about the status change.
+
+    Args:
+        recipient_list: list of str. A list of recipient id
```

Comment:
Nits: id --> ids; end with full stop for consistency. Add full stops below, too.